Add entity class limit explanation to entities and license references (#2435)
sensu/sensu-docs: content/sensu-go/5.20/reference/entities.md

If your Sensu instance includes more than 100 entities, [contact us][10] to learn how to upgrade your installation and increase your limit.
See [the announcement on our blog][11] for more information about our usage policy.

> Commercial licenses include an entity limit and an entity class limit:

Reviewer: Entity limit and entity class limit are not required. If undefined, the limit does not exist. Multiple entity class limits can exist.
```suggestion
Commercial licenses may include an entity limit and entity class limits:
```

- Entity limit: the maximum number of entities of all classes your license includes. Both agent and proxy entities count toward the overall entity limit.

> - Entity class limit: the maximum number of agent entities your license includes.

Reviewer: ```suggestion
- Entity class limits: the maximum number of entities of each entity class your license includes.
```

> - Entity class limit: the maximum number of agent entities your license includes.

Author: ```suggestion
- Entity class limit: the maximum number of a specific class of entities (e.g. agent or proxy) that your license includes.
```

For example, if your license has an entity limit of 10,000 and an agent entity class limit of 3,000, you cannot run more than 10,000 entities (agent and proxy) total.
At the same time, you cannot run more than 3,000 agents.
If you use only 1,500 agent entities, you can have 8,500 proxy entities before you reach the overall entity limit of 10,000.

> Use sensuctl or the license API to [view your entity counts and limits][29].

Author: ```suggestion
Use sensuctl or the license API to [view your overall entity count and limit][29].
Your entity class limits are listed in your YAML/JSON Sensu license.
To find your entity class counts, query your store or sensuctl with a tool like [jq][30].
```

If your Sensu instance includes more than 100 entities, [contact us][10] to learn how to upgrade your installation and increase your limit.
See [the announcement on our blog][11] for more information about our usage policy.

Commercial licenses may include an entity limit and entity class limits:

- Entity limit: the maximum number of entities of all classes your license includes. Both agent and proxy entities count toward the overall entity limit.

> - Entity class limit: the maximum number of a specific class of entities (e.g. agent or proxy) that your license includes.

Reviewer: ```suggestion
- Entity class limits: the maximum number of a specific class of entities (e.g. agent or proxy) that your license includes.
```

For example, if your license has an entity limit of 10,000 and an agent entity class limit of 3,000, you cannot run more than 10,000 entities (agent and proxy) total.
At the same time, you cannot run more than 3,000 agents.
If you use only 1,500 agent entities, you can have 8,500 proxy entities before you reach the overall entity limit of 10,000.

Use sensuctl or the license API to [view your overall entity count and limit][29].

> Your entity class limits are listed in your YAML/JSON Sensu license.
> To find your entity class counts, query your store or sensuctl with a tool like jq.

Author: ```suggestion
```

> To find your entity class counts, query your store or sensuctl with a tool like [jq][30].

Author: ```suggestion
To find your entity class counts, query your store or sensuctl with a tool like jq.
```

> To find your entity class counts, query your store or sensuctl with a tool like [jq][30].

Reviewer: IMO we shouldn't document workarounds for features that don't exist yet. I think the issue history is sufficient enough for this. Might be best to remove these lines.

[26]: #processes-attributes
[27]: ../agent/#discover-processes
[28]: http://man7.org/linux/man-pages/man1/top.1.html

> [29]: ../license/#view-entity-count-and-entity-limit

Author: ```suggestion
[29]: ../license/#view-entity-count-and-entity-limit
[30]: https://stedolan.github.io/jq/
```

Reviewer: Oh I'm not sure I would link `jq` here in our docs.

[26]: #processes-attributes
[27]: ../agent/#discover-processes
[28]: http://man7.org/linux/man-pages/man1/top.1.html
[29]: ../license/#view-entity-count-and-entity-limit

> [30]: https://stedolan.github.io/jq/

Author: ```suggestion
```

> [30]: https://stedolan.github.io/jq/

Reviewer: ```suggestion
```
